eslint.config.mjs

```diff
@@ -0,0 +1,41 @@
+import bpmnIoPlugin from 'eslint-plugin-bpmn-io';
+
+const files = {
+  build: [
+    '*.js',
+    '*.mjs'
+  ],
+  test: [
+    'test/**/*.js'
+  ]
+};
+
+export default [
+
+  // build
+  ...bpmnIoPlugin.configs.node.map(config => {
```

Comment:
Taking this configuration we don't validate [`rules`](https://github.com/camunda/bpmnlint-plugin-camunda-compat) at all, please add a lib part to the configuration. You can introduce a bug under _rules_ and check if validation breaks (which would be the expected behavior).